src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
```

Comment:
There should not be an empty line before the method definition.

Reply:
I think that is also incorrect, I dont think the methods should be stacked against each other line by line